video/bot/render.yaml

```
@@ -0,0 +1,14 @@
+    name: telegram-video-bot
+    runtime: python
+    buildCommand: pip install -r requirements.txt
+    startCommand: python main.py
+    env: python
+    envVars:
+      - key: BOT_TOKEN
+        value: 8006332215:AAH-xXrm8fmPBEC9GcdWiVErnt3Fr1TL_DU
+    plan: free
+    region: singapore
+    autoDeploy: true
+    healthCheckPath: /health
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)